clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -5445,17 +5445,17 @@ void Sema::AddIntelFPGABankBitsAttr(Decl *D, const AttributeCommonInfo &CI,
                  IntelFPGABankBitsAttr(Context, CI, Args.data(), Args.size()));
 }
 
-static void handleIntelFPGAMaxPrivateCopiesAttr(Sema &S, Decl *D,
+static void handleIntelFPGAPrivateCopiesAttr(Sema &S, Decl *D,
                                               const ParsedAttr &Attr) {
```

Comment:
I believe clang-format should be applied for the commit.

Reply:
Fixed.